components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -7,26 +7,79 @@ documented [here](https://mozilla.github.io/glean_parser/metrics-yaml.html).
 
 When adding a new metric, the workflow is:
 * Decide on which metric type you want to use.
-* Add a new entry to metrics.yaml.
-* Add code to your project to record into the metric by calling Glean.
+* Add a new entry to `metrics.yaml`.
+* Add code to your project to record into the metric by calling glean.
 
 **Important**: as stated [here](../../README.md#before-using-the-library), any new data collection requires
 documentation and data-review. This is also required for any new metric automatically collected
 by glean.
 
+# The `metrics.yaml` file
+
+The `metrics.yaml` file defines the metrics your application or library will
+send. They are organized into categories. Category names may optionally contain
+`.`'s to indicate subcategories. 
```

Comment:
Nit: drop the subcategories here.

Reply:
I find that feature *really* surprising, and I don't think there's another way to discover it if we don't document it.

Reply:
But if you show it in the example, is that not sufficient?
But either way i'm fine with this.